Readability.js

```diff
@@ -131,6 +131,7 @@ Readability.prototype = {
     prevLink: /(prev|earl|old|new|<|«)/i,
     whitespace: /^\s*$/,
     hasContent: /\S$/,
+    srcsetUrl: /(\S+)(\s+[\d.]+[xw])?(\s*,\s*)?/g,
```

Comment:
> can we make the comma part non-optional

It's done in [6c13872](https://github.com/mozilla/readability/pull/590/commits/6c1387234b5c69ed6d86a5574a96ee00558a42ef).

> there is now less whitespace in the NY times testcase, for instance (the whitespace after the `,` got dropped)

Actually, the less whitespace is the correct one. In fact that's how it is currently in master. The whitespace after the `,` was added because I was using `matchAll` methods to get array of new sources, which then combined using `join(", ")` as can be seen [here](https://github.com/mozilla/readability/blob/4fd18f40d17b6b7ed23e770ff92eca5c0be011c6/Readability.js#L400).